CHANGELOG.md

```diff
@@ -9,6 +9,29 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 The following changes have been implemented but not released yet:
 
+### Bugfix
+
+#### node
+
+- The session expiration date (`session.info.expirationDate`) is now correct when loading a `Session` from storage.
+
+### Feature
+
+#### node
+
+- It is now possible to use load a function using `getSessionFromStorage` and not log it in
```

Comment:
This sentence doesn't make sense to me - can you look at the wording; "to use load a function"

Reply:
Oops, that's a Frankenstein sentence that got rewritten a couple of times, and eventually got rebuilt wrong. Fixed in https://github.com/inrupt/solid-client-authn-js/pull/3775/commits/656b8d40b61f0b48529adbce95619ea0b0602f9b